test/unit/dns_powerdns_configuration_validator_test.rb

```diff
@@ -0,0 +1,85 @@
+require 'test_helper'
+
+require 'smart_proxy_dns_powerdns/dns_powerdns_plugin'
+require 'smart_proxy_dns_powerdns/dns_powerdns_configuration_validator'
+
+class DnsPowerdnsConfigurationValidatorTest < Test::Unit::TestCase
+  def setup
+    @config_validator = Proxy::Dns::Powerdns::ConfigurationValidator.new
+  end
+
+  def test_initialize_missing_backend
+    Proxy::Dns::Powerdns::Plugin.load_test_settings(
```

Comment:
Good idea. That would also solve the problem that load_test_settings doesn't tear down.